README.md

```diff
@@ -17,18 +17,15 @@ The [official CocoaPods website](http://cocoapods.org) has detailed instructions
 The following lines in a Podfile will install the SDK for an iOS app with a watch OS 2 extension: 
 
 ```
-source 'https://github.com/openwhisk/openwhisk-podspecs.git'
-
+install! 'cocoapods', :deterministic_uuids => false
```

Comment:
This is a new one I was not aware of https://guides.cocoapods.org/syntax/podfile.html#install_bang
Sorry for my ignorance but does the `:deterministic_uuids => false` do for us?

Using this requires a minimum version of cocoapod cli that we need to let folks know about?